pyproject.toml

```diff
@@ -63,12 +63,12 @@ name = "Miscellaneous internal changes"
 showcontent = false
 
 [tool.poetry.dependencies]
-python = ">=3.6,<3.9"
+python = ">=3.7,<3.9"
 coloredlogs = ">=10,<16"
-sanic = ">=19.12.2,<21.0.0"
-sanic-cors = "^0.10.0"
+sanic = "^21.6.0"
 requests = ">=2.23.0,<2.26.0"
 typing-extensions = "^3.7.4"
+Sanic-Cors = "^1.0.0"
```

Comment:
could you keep this lowercase though? it seems that both would work

Reply:
That's what I got runnning `poetry add sanic-cors=^1.0.0` I also find it very odd. Yet, pypi also lists the package with those capatalized letters. https://pypi.org/project/Sanic-Cors/

Reply:
🤷🏻 magic